config/api.yml

```diff
@@ -42,6 +42,20 @@
     - :subcollection
     :verbs: *g
     :klass: Account
+  :ansible_repositories:
+    :description: Ansible Repositories
```

Comment:
looking at the exposed class below, are these repositories or playbooks/scripts ?  i.e. /api/ansible_scripts or /api/ansible_playbooks maybe ?